extensions/stackdriver/log/logger.cc

```diff
@@ -162,6 +162,7 @@ void Logger::addLogEntry(const ::Wasm::Common::RequestInfo& request_info,
       std::string(::Wasm::Common::AuthenticationPolicyString(
           request_info.service_auth_policy));
   (*label_map)["protocol"] = request_info.request_protocol;
+  (*label_map)["filtered"] = request_info.sampled ? "True" : "False";
```

Comment:
nit: could you make it lower case? All other labels are lower cased except enum ones.

Reply:
should we be consistent with terminology here?  if the request is selected for logs-sampling, should this just be `sampled` ?

Reply:
I myself was thinking the same... as PAY as you Go(https://cloud.google.com/pricing) team has not implemented "filtered" label yet.. changing it to sampled...